web-client/integration-tests/journey/petitionsClerkAddsNoticeToCase.js

```diff
@@ -43,6 +43,13 @@ export const petitionsClerkAddsNoticeToCase = test => {
     expect(test.getState('validationErrors')).toEqual({});
     expect(test.getState('pdfPreviewUrl')).toBeDefined();
 
+    // await test.runSequence('setPDFSignatureDataSequence', {
```

Comment:
@ericsorenson Why is this commented out?

Reply:
It was removed as part of a story, I'm adding a TODO with details.